lib/Minz/Log.php

```diff
@@ -20,6 +20,8 @@ class Minz_Log {
 	const NOTICE = 8;
 	const DEBUG = 16;
 
+	const MAX_LOG_SIZE = 512000; // 500kB
```

Comment:
Please put a define in `constants.php` and here, something like
```php
if (!defined('MAX_LOG_SIZE')) {
	define('MAX_LOG_SIZE', 512000);
}
```

Reply:
Done